docs/it-manual/sre-playbook/monitoring.md

```diff
@@ -130,4 +132,4 @@ In the Configuration section of the service, you can see current Auto Scaling po
 
 By default, there is a high and low CPU auto-scaling policy, which adds or removes a task if the CPU is higher / lower than the alarm thresholds (mentioned above).
 
-You can change the task and memory sizes by updating the variables `ECS_TASK_CPU` and `ECS_TASK_MEMORY` in the civiform_config.sh file of the forked civiform-deploy repository.
\ No newline at end of file
+You can change the task and memory sizes by updating the variables `ECS_TASK_CPU` and `ECS_TASK_MEMORY` in the civiform_config.sh file of the forked civiform-deploy repository.
```

Comment:
Is there a change on this line?

Reply:
I'm not sure why that shows as a diff, but there isn't